solidity/contracts/converter/BancorConverter.sol

```diff
@@ -56,10 +56,10 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
     /**
       * @dev triggered when a conversion between two tokens occurs
       * 
-      * @param _fromToken       ERC20 token converted from
-      * @param _toToken         ERC20 token converted to
+      * @param _fromToken       source ERC20 token

```

Comment:
@yudilevi : Did you miss these two, or would it break current API?

Reply:
same - will break the API